docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -222,12 +593,60 @@ The following table lists all the server state attributes the client can receive
+| `MC_PLAYBACK_STATE_FAST_FORWARDING` | Playback state of fast forwarding (Tizen 4.0 and higher for Mobile and Tizen 5.0 and Higher for Wearable) |
+| `MC_PLAYBACK_STATE_REWIND`       | Playback state of rewinding (Tizen 4.0 and higher for Mobile and Tizen 5.0 and Higher for Wearable) |
+
+<a name="playbackaction"></a>
+## Media Controller Playback Action Attributes
+
+The following table lists all the playback action attributes the client can send command and the server can receive.
+
+**Table: Media controller playback action attributes**
+
+| Attribute                        | Description                              |
+|----------------------------------|------------------------------------------|
+| **Playback actions**             |                                          |
+| `MC_PLAYBACK_ACTION_PLAY`        | Playback action of play                  |
+| `MC_PLAYBACK_ACTION_PAUSED`      | Playback action of pause                 |
+| `MC_PLAYBACK_ACTION_STOPPED`     | Playback action of stop                  |
+| `MC_PLAYBACK_ACTION_NEXT`        | Playback action of moving to next media  |
+| `MC_PLAYBACK_ACTION_PREV`        | Playback action of moving to previous media |
+| `MC_PLAYBACK_ACTION_FAST_FORWARD` | Playback action of fast forward         |
+| `MC_PLAYBACK_ACTION_REWIND`      | Playback action of rewind                |
+| `MC_PLAYBACK_ACTION_TOGGLE_PLAY_PAUSE` | Playback action of toggle between play and pause |
+
+> **Note**
+>
+> This Attributes support Tizen 4.0 and higher for Mobile.
+
+<a name="shufflemode"></a>
+## Media Controller Shuffle Mode Attributes
+
+The following table lists all the shuffle mode attributes the client can receive and send command.
+
+**Table: Media controller shuffle mode attributes**
+
+| Attribute                        | Description                              |
+|----------------------------------|------------------------------------------|
+| **Shuffle modes**                |                                          |
 | `MC_SHUFFLE_MODE_ON`             | Shuffle mode is on                       |
 | `MC_SHUFFLE_MODE_OFF`            | Shuffle mode is off                      |
+
+<a name="repeatmode"></a>
+## Media Controller Repeat Mode Attributes
+
+The following table lists all the repeat mode attributes the client can receive and send command.
+
+**Table: Media controller repeat mode attributes**
+
+| Attribute                        | Description                              |
+|----------------------------------|------------------------------------------|
 | **Repeat mode states**           |                                          |
 | `MC_REPEAT_MODE_ON`              | Repeat mode is on                        |
 | `MC_REPEAT_MODE_OFF`             | Repeat mode is off                       |
+| `MC_REPEAT_MODE_ONE_MEDIA`       | Repeat mode is on for one media (Tizen 4.0 and higher for Mobile and Tizen 5.0 and Higher for Wearable) |
 
 <a name="servermetadata"></a>
```

Comment:
As mentioned earlier, please avoid using HTML code. Use only MD specific code.

Reply:
The 'servermetadata' tag is not used. So I removed it.